barbican/cmd/barbican_manage.py

```diff
@@ -344,10 +347,38 @@ def _verify_label_does_not_exist(self, key_type, label, session):
             print("The label {label} already exists!".format(label=label))
             sys.exit(1)
 
+class SAPCommands(object):
+
+    description = "Move all secrets associated with the old_project_id to the new_project_id."
+
+    @args('--db-url', '-d', metavar='<db-url>', dest='dburl',
+          help='Barbican database URL')
+    @args('--old-project-id', '-o', metavar='<old-project-id>', dest='old_project_id',
+          help='The old project ID to move secrets from.')
+    @args('--new-project-id', '-n', metavar='<new-project-id>', dest='new_project_id',
+          help='The new project ID to move secrets to.')
+    @args('--verbose', '-V', action='store_true', dest='verbose',
+          default=False, help='Show full information about the secret movement.')
+    def move_secrets(self, conf, dburl=None, old_project_id=None, new_project_id=None, verbose=None):
+        if dburl is None:
+            dburl = conf.sql_connection
+
+        engine = create_engine(dburl)
+        with engine.connect() as connection:
+            try:
+                query = text("UPDATE secrets SET project_id = :new_project_id WHERE project_id = :old_project_id")
+                connection.execute(query, new_project_id=new_project_id, old_project_id=old_project_id)
+                print("All secrets associated with old project ID moved to new project ID.")
+                if verbose:
+                    print(f"Moved secrets from project ID {old_project_id} to {new_project_id}.")
```

Comment:
Why do we need 2 prints? If we always have a print, then I'd keep the second one over the first one.

Reply:
if we enable Verbose the second print works.

Reply:
I know, but does the first one even help? Why do we need a verbose mode anyways, if the only change is printing the project ids, which would make sense in any case?